Project.toml

```
@@ -29,6 +31,7 @@ GitHub = "5.2"
 HTTP = "0.8, 0.9.1, 1"
 JSON = "0.19, 0.20, 0.21"
 LicenseCheck = "0.2"
+PackageAnalyzer = "1"
```

This will let us install a dummy version of the package (registered at v0.2) on Julia 1.3 to 1.5:
```suggestion
PackageAnalyzer = "0.2, 1"
```
xref https://github.com/JuliaRegistries/General/pull/76413